docs/reference/subnet-evm/API-subnet-evm.md

```diff
@@ -1,13 +1,16 @@
 ---
-sidebar_position: 14
+id: api
+tags: [Subnet-EVM, AvalancheGo APIs]
```

Comment:
Why is this tagged AvalancheGo APIs?

Reply:
good catch, was a copy paste error. Originally has subnet-evn under the "Avalanche-Go" section, but moved it to reflect your suggestion in the mural